include/fi_proto.h

```diff
@@ -56,6 +56,8 @@ enum {
 	ofi_ctrl_data,
 	ofi_ctrl_large_data,
 	ofi_ctrl_ack,
+	ofi_ctrl_nack,
+	ofi_ctrl_discard,
```

Comment:
Hmm, I think we can keep the version as it is, since none of the providers use this currently.